### ISSUE TITLE
Allow separate windows to have individual zoom settings

### ISSUE THREAD

**Issue by dluc** [microsoft/vscode#72011]
Sorry to open a new item but https://github.com/Microsoft/vscode/issues/24226 has been locked, any chance this feature request could be revisited and addressed without the need to create workspaces (which requires editing a file instead of using shortcuts)?

I don't use workspaces, I just enter a folder from a terminal and type `code .` to work on the folder, which could be part of a project, a full project or not.

I work with multiple monitors and often present code at meetings, either mirroring a laptop screen or not, and the global zoom settings makes this task difficult, particularly when screens have different DPIs/scaling. E.g. I want to zoom only on one window to some huge font and back to small, as quickly as possible using the keyboard, without affecting other windows, without having to edit settings in the middle of a presentation.

Thanks

**Comment by jsargeim** [comment]
Updates? This would be awesome.

**Comment by awolant** [comment]
@dluc It does not solve the issue completely, but you might find `Editor: Font Zoom In/Out` useful. It works per window. It zooms only the code in the editor. Rest of the window stays the same. You can bind it to a shortcut and have something similar to what you wanted.

**Comment by MeTaNoV** [comment]
This is indeed really annoying, and I am looking forward this issue to be reopened...

@awolant, thanks for the hint, I will give it a try!

**Comment by JonKDahl** [comment]
I am surprised to hear that this basic functionality is not available in vscode. I hope this will be reconsidered.

Thanks for the good tip, @awolant

**Comment by wxupjack** [comment]
> I am surprised to hear that this basic functionality is not available in vscode. I hope this will be reconsidered.
> 
> Thanks for the good tip, @awolant

Now I found a way to do this :

1. open setting, find **_Window: Zoom Level_**
2. switch to **_workspace_** tab rather than **_user_**
3. change to any other value, and then try **_ctrl/cmd + -_** ✌️

Because it will add `"window.zoomLevel": 0` in this workspace `./.vscode/setting.json` file (or you can directly add this line in setting.json).
VS code will not change the window zoom level if the workspace has its own setting. #

**Comment by oshihirii** [comment]
I thought a solution had been implemented as I saw this setting which enabled zoom in a single window.  

![vs_code_mouse_wheel_zoom](https://user-images.githubusercontent.com/5090307/142819813-69b225ac-471d-42ad-afe0-d6636bf0c31e.png)

However, I use multiple VS Code windows on different monitors.

To open a new window, I press <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>N</kbd>, and then drag it to another monitor.  

When I open `Settings` in the *original* window, I have the option of switching between `User` and `Workspace` tabs.

And I checked the above `Mouse Wheel Zoom` option in the `Workspace` tab there.  

However, when I open `Settings` in the *second* window, I only have the `User` tab available.

Furthermore, the scroll/zoom behaviour does not occur in that second window.  

The solution for me was to check the `Mouse Wheel Zoom` feature on the `User` tab in the *original* window, and then I could zoom my different windows independently of each other.  

![vs_code_mouse_wheel_zoom_user](https://user-images.githubusercontent.com/5090307/142820887-e2fd34bf-c8f0-49a0-9338-3e9fb0e22d02.png)

This is only my first day using VS code, so not sure if this is best practise or not.
